broadinstitute/single_cell_portal_core
# adding override to annotation viz limit values (SCP-4372)

This adds a new study default options field 'override_viz_limit_annotations' which is an array of annotations to show regardless of whether they exceed the viz limit threshold.  This will be needed for the U19 study.  Right now, this property is not accessible via the UX -- it's complex and rare enough that I think it makes the most sense to make it require SCP support to set.  Obviously, if this becomes a frequent request, we can look at the best place to expose it in our UX, but for now I don't think it would be worth either the effort or the extra complexity this would bring to the upload UX.  

TO TEST:
1. populate the new synthetic study `SyntheticStudyPopulator.populate('mouse_multilabel')`
2.  Observe that the 'manylabel_field' is not visualizable in the explore tab
3. In the rails console, run 
 ```
    s = Study.find_by(accession: '<<accession of the new study')
    s.add_override_viz_limit_annotation('manylabel_field')
```
4. After that operation completes, confirm that the 'manylabel_field' is now visualizable in the explore tab
